source/fuzz/transformation_swap_function_variables.h

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2021 Mostafa Ashraf
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SOURCE_FUZZ_TRANSFORMATION_SWAP_FUNCTION_VARIABLES_H_
+#define SOURCE_FUZZ_TRANSFORMATION_SWAP_FUNCTION_VARIABLES_H_
+
+namespace spvtools {
+namespace fuzz {
+
+#include "source/fuzz/protobufs/spirvfuzz_protobufs.h"
+#include "source/fuzz/transformation.h"
+#include "source/fuzz/transformation_context.h"
+#include "source/opt/ir_context.h"
+
+// Transformation that's responsible for single change on just one method
+class TransformationSwapFunctionVariables : public Transformation {
+ public:
+  explicit TransformationSwapFunctionVariables(
+      const protobufs::TransformationSwapFunctionVariables& message);
+
+  TransformationSwapFunctionVariables(std::pair<uint32_t, uint32_t> pair_id,
+                                      uint32_t function_id, uint32_t fresh_id);
+
+  bool IsApplicable(
```

Comment:
Please add a comment explaining the conditions of IsApplicable. Take a look at some other transformation classes for ideas.